course7/homeworks/patrickkostjens/ls/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+cmake_minimum_required(VERSION 3.10)
```

Comment:
You are not using any CMake features introduced in 3.10. It's better to use lower number here, the lowest possible.
3.0 is a reasonable value.
Otherwise, people with slightly older version won't be able to use it!